tests/test_security.py

```diff
@@ -99,43 +100,193 @@ async def test_show_user(registered_client, registered_superuser_client):
     assert res.status_code == 200
 
 
-async def test_edit_user(registered_client, registered_superuser_client):
+async def test_edit_current_user(registered_client, app):
 
-    res = await registered_superuser_client.post(
-        f"{PREFIX}/register/",
-        json=dict(email="to_patch@asd.asd", password="12"),
+    res = await registered_client.get(f"{PREFIX}/current-user/")
+    expected_status = res.json()
+
+    res = await registered_client.patch(f"{PREFIX}/current-user/", json={})
+    assert res.status_code == 200
+    assert res.json() == expected_status
+
+    # CACHE_DIR
+
+    res = await registered_client.patch(
+        f"{PREFIX}/current-user/", json={"cache_dir": "/tmp"}
     )
-    user_id = res.json()["id"]
-    assert res.status_code == 201
+    assert res.status_code == 200
+    expected_status["cache_dir"] = "/tmp"
+    assert res.json() == expected_status
 
-    # PATCH/{user_id} with non-superuser user
+    # From val_absolute_path:
+    # - String attribute 'cache_dir' cannot be empty
     res = await registered_client.patch(
-        f"{PREFIX}/users/{user_id}/", json={"slurm_user": "my_slurm_user"}
+        f"{PREFIX}/current-user/", json={"cache_dir": ""}
     )
-    assert res.status_code == 403
+    assert res.status_code == 422
+    # - String attribute 'cache_dir' must be an absolute path (given 'not_abs')
+    res = await registered_client.patch(
+        f"{PREFIX}/current-user/", json={"cache_dir": "not_abs"}
+    )
+    assert res.status_code == 422
+    # - String attribute '{attribute}' cannot be None
+    res = await registered_client.patch(
+        f"{PREFIX}/current-user/", json={"cache_dir": None}
+    )
+    assert res.status_code == 422
+
+    # PASSWORD
```

Comment:
split cache_dir and password tests, in view of upcoming PR about password update